packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -274,6 +274,20 @@ export default async function pluginContentBlog(
             },
           });
 
+          if (metadata.aliases) {
+            metadata.aliases.forEach((alias) => {
+              addRoute({
+                path: alias,
+                component: blogPostComponent,
```

Comment:
IIUC, this would create two pages with the same content? Isn't it simpler to set up redirects instead?

Reply:
Hey @Josh-Cena,

The context for this is in @slorber's comment here: https://github.com/facebook/docusaurus/discussions/8311#discussioncomment-4168093

> ```md
> ---
> slug: /2022/11/17/azure-ad-claims-static-web-apps-azure-functions
> aliases: ["/"] # NEW FEATURE
> ---
> 
> # Azure AD Claims with Static Web Apps and Azure Functions
> 
> Text
> ```
> 
> The aliasing feature would create extra pages for another slug (that can be /)
>
> It gives the ability to keep a shared canonical URL for both pages and avoid SEO content being flagged as duplicate.

Does this make sense?

Reply:
yes that makes sense to me, and we'll likely want to add something similar to individual docs for consistency